src/utils/mod.rs

```diff
@@ -0,0 +1,51 @@
+use std::path::PathBuf;
+
+use clap::Subcommand;
+
+use self::vec3::{Float, Vec3};
+
+pub mod stl_generator;
+pub mod stl_parser;
+mod vec3;
+
+#[derive(Debug, Clone, Copy, clap::ValueEnum)]
+pub enum STLFormat {
```

Comment:
use camel case naming style:

```suggestion
pub enum StlFormat {
```

usually, abbreviations are also formatted in camel case. for [example](https://github.com/Devolutions/sspi-rs/blob/master/src/credssp/mod.rs#L81):

```rust
pub enum CredSspMode { // <----. not `CredSSPMode`
    // ...
}
```